response/slack/signals.py

```diff
@@ -28,6 +29,22 @@ def update_headline_after_incident_save(sender, instance, **kwargs):
         )
 
 
+@receiver(post_save, sender=Incident)
+def prompt_incident_report(sender, instance: Incident, **kwargs):
+    """
+    Prompt incident lead to complete a report when an incident is closed.
+    """
+
+    if instance.is_closed():
+        user_to_notify = instance.lead or instance.reporter
+        doc_url = urljoin(
```

Comment:
This could refactored into an model function, but it's only used here and to update the slack topic.